Makefile

```diff
@@ -32,12 +32,25 @@ DEADLOCK_DISABLE := $$(\
 						find . -name "*.bak" | xargs rm && \
 						go mod tidy)
 
+BUILD_FLAGS ?=
 BUILD_TAGS ?=
+BUILD_CGO_ENABLED := 0
 
 ifneq ($(SWAGGER), 0)
 	BUILD_TAGS += swagger_server
 endif
 
+ifeq ($(DASHBOARD), 0)
+	BUILD_TAGS += without_dashboard
+else
+	BUILD_CGO_ENABLED := 1
+endif
```

Comment:
how about
```
ifneq ($(DASHBOARD), 0)
  BUILD_TAGS += with_dashboard
  BUILD_CGO_ENABLED := 1
endif
```

Reply:
Now, our test code is without tags, in order to ensure that the dashboard will be started in this case.

Reply:
can we add `DASHBOARD=1` when run `go test`?

Reply:
It is useless to add this when running `go test`, this is only useful in the `Makefile`. `go test` needs to add `-tags "with_dashboard"`. I think the current method has changed a little, and the default startup also meets the current requirements.

Reply:
ok